.travis.yml

```diff
@@ -35,10 +35,8 @@ matrix:
     - language: android
       android:
         components:
-          - build-tools-28.0.3
-          - android-28
-          - android-24
```

Comment:
No wonders it broke. You removed emulator image from components 🙊

Reply:
nah man it's broken on master

Reply:
Yeah, I figured it out later. Tried to fix this (look at commits below) but sadly nothing works. May look at this at a later date (like tomorrow)

Reply:
thanks for trying ❤️ 

fucking travis.

maybe it's time to look at other RN projects, see how they handle CI? And maybe ditch Travis in favor of Circle?